mappings/net/minecraft/util/math/Quaternion.mapping

```diff
@@ -1,5 +1,10 @@
 CLASS net/minecraft/class_1158 net/minecraft/util/math/Quaternion
+	COMMENT Quaternion is a type of number used to represent a counterclockwise
+	COMMENT rotation around an axis.
```

Comment:
Not seeing these show up in the tiny file. and why are there 2?

Reply:
You mean you tried building it yourself?

Reply:
Yes

Reply:
uhh i'm gonna need to take a look at it then

Reply:
I'm not sure what you mean by "counterclockwise rotation", a quaternion isn't stored as an axis plus an angle. The angle of rotation is the angle of the vector (sqrt(b^2 + c^2 + d^2), a).

Reply:
There are two because there are new lines. I intentionally designed it to be before the commented element and swap on new lines so that it will be easier for pr reviews.

Rune: the counterclockwise rotation means that the constructor that takes a vec3 and an angle is like a rotation counterclockwise around that vec3 axis for that much angle.

Reply:
Removed for now. Since modmuss points out jd still has some issues, we can delay that a bit and keep them in github issues for now.